SourceViewVC/SourceViewVC.php

```diff
@@ -0,0 +1,169 @@
+<?php
+
+# Copyright (c) 2015 John Bailey
+# Copyright (c) 2012 John Reese
+# Licensed under the MIT license
+
+if ( false === include_once( config_get( 'plugin_path' ) . 'SourceSVN/SourceSVN.php' ) ) {
+	return;
+}
+
+class SourceViewVCPlugin extends SourceSVNPlugin {
+	public function register() {
+		$this->name = lang_get( 'plugin_SourceViewVC_title' );
+		$this->description = lang_get( 'plugin_SourceViewVC_description' );
+
+		$this->version = '0.1';
+		$this->requires = array(
+			'MantisCore' => '1.2.0',
+			'Source' => '0.16',
+			'SourceSVN' => '0.16',
+		);
+
+		$this->author = 'John Bailey';
+		$this->contact = 'dev@brightsilence.com';
+		$this->url = 'https://github.com/bright-tools/source-integration';
```

Comment:
Is it intentional to have the URL point to your personal fork ? IMHO it would make more sense to link to mantisbt-plugins instead.